build.gradle

```diff
@@ -12,6 +12,11 @@ allprojects {
   group = 'com.salesforce.androidsdk'
   
   repositories {
+    maven {
+      // All of React Native (JS, Android binaries) is installed from npm
```

Comment:
Starting in 0.21, react native is no longer on maven central.
Instead we need to get the npm package and point to it from the build.gradle file.